sql_generators/active_users/__init__.py

```diff
@@ -55,6 +55,18 @@ def generate(target_project, output_dir, use_cloud_function):
     focus_android_view_template = env.get_template("focus_android_view.sql")
     mobile_view_template = env.get_template("mobile_view.sql")
     output_dir = Path(output_dir) / target_project
+    view_columns = (
+        "\t* EXCEPT(app_version),\n\tapp_version,\n\t"
```

Comment:
I think there would still be value to keep the `app_version` field.  Similarly the full os version is also kept [here](https://github.com/mozilla/bigquery-etl/blob/a2c1655abef2615ba548f2df34e905ff865949be/sql_generators/active_users/templates/desktop_query.sql#L16).